ui/js/directives/treeherder/main.js

```diff
@@ -151,65 +137,3 @@ treeherder.directive('bugInput', function () {
         }
     };
 });
-
-treeherder.directive('numbersOnly', function () {
-    return {
-        require: 'ngModel',
-        link: function (scope, element, attrs, modelCtrl) {
-            modelCtrl.$parsers.push(function (inputValue) {
-                // this next is necessary for when using ng-required on your input.
-                // In such cases, when a letter is typed first, this parser will be called
-                // again, and the 2nd time, the value will be undefined
-                if (inputValue === undefined) return '';
-                var transformedInput = inputValue.replace(/[^0-9]/g, '');
-                if (transformedInput!==inputValue) {
-                    modelCtrl.$setViewValue(transformedInput);
-                    modelCtrl.$render();
-                }
-
-                return transformedInput;
-            });
-        }
-    };
-});
-
-treeherder.directive("thTruncatedList",
```

Comment:
Unused as of https://github.com/mozilla/treeherder/commit/83c94d9c#diff-2f26aabb7ebf8e56afda0c05f366b04fL34